vwb/app/controllers/userlogins/omniauth_callbacks_controller.rb

```diff
@@ -19,7 +19,13 @@ def after_omniauth_failure_path_for(_scope)
     end
   
     def after_sign_in_path_for(resource_or_scope)
-      stored_location_for(resource_or_scope) || root_path
+      #stored_location_for(resource_or_scope) || root_path
+      if User.exists?(email: current_userlogin.email)
+        # request.env['omniauth.origin'] || stored_location_for(resource) || root_url
+        root_url
+      else
+        registration_user_path

```

Comment:
Goes to registration if user has not submitted additional information. Looks good